### PR TITLE
docs: moved SimNet guide into SimNet-Guide.md ...

### PR DESCRIPTION
**Pull request by kilrau**
…and referenced it in Exchange-Guide.md

### CODE REVIEW

File: SimNet-Guide.md [hunk 1]
```diff
@@ -0,0 +1,227 @@
...
+sudo ln -s /usr/lib/go-1.10/bin/go /usr/local/bin/go 
+```
+
+### Repository Checkout
+
+```
+git clone https://github.com/ExchangeUnion/xud-simnet.git ~/xud-simnet
+```
+
+This should create the `xud-simnet` directory under your home directory.
```
Comment (michael1011) [inline SimNet-Guide.md]: I would rather say `in your home directory`.
```suggestion
This should create the `xud-simnet` directory in your home directory.
```

File: SimNet-Guide.md [hunk 2]
```diff
@@ -0,0 +1,227 @@
...
+
+### Setup ~/.bashrc
+
+To enable access to the scripts please update and source your `.bashrc`:
+
+```
+source ~/xud-simnet/setup.bash
+source ~/.bashrc
```
Comment (michael1011) [inline SimNet-Guide.md]: I would run `source ~/xud-simnet/setup.bash` after `source ~/.bashrc` to make sure the simnet environment variables override the default ones of the user

File: SimNet-Guide.md [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+```
+
+Please note that the `setup.bash` script set your `$GOPATH` to `~/xud-simnet/go`. All changes from `setup.bash`are temporary and only active for the current terminal session. You can run
```
Comment (michael1011) [inline SimNet-Guide.md]:
```suggestion
Please note that the `setup.bash` script sets your `$GOPATH` to `~/xud-simnet/go`. All changes from `setup.bash`are temporary and only active for the current terminal session. You can run
```

File: Exchange-Guide.md [hunk 4]
```diff
@@ -2,198 +2,14 @@
...
-
-This should create the `xud-simnet` directory under your home directory.
-
-### Setup ~/.bashrc
-
-To enable access to the scripts please update and source your `.bashrc`:
-
-```
-source ~/xud-simnet/setup.bash
-source ~/.bashrc
```
Comment (dopetard) [inline Exchange-Guide.md]: Maybe `source ~/.bashrc` should run before `source ~/xud-simnet/setup.bash`?

File: SimNet-Guide.md (hunk 1 above)
Comment (kilrau) [reply SimNet-Guide.md]: Good catch, fixed.

File: SimNet-Guide.md (hunk 2 above)
Comment (kilrau) [reply SimNet-Guide.md]: Fixed.

File: SimNet-Guide.md (hunk 3 above)
Comment (kilrau) [reply SimNet-Guide.md]: Fixed.

File: Exchange-Guide.md (hunk 4 above)
Comment (kilrau) [reply Exchange-Guide.md]: Already fixed.

**Pull request merged**